x-pack/plugins/endpoint/scripts/event_mapping.json

```diff
@@ -386,7 +386,8 @@
                 "type": "nested"
               },
               "file_extension": {
-                "type": "long"
+                "ignore_above": 1024,
+                "type": "keyword"
```

Comment:
❔ this one too? 
https://github.com/elastic/kibana/blob/c109f55b354b14796f9719b2a2ebf4d9aff24e21/x-pack/plugins/endpoint/scripts/alert_mapping.json#L396